indra/newview/llconversationview.cpp

```diff
@@ -578,7 +578,12 @@ participant_id(),
 avatar_icon("avatar_icon"),
 info_button("info_button"),
 output_monitor("output_monitor")
-{}
+{
+    if (!output_monitor.visible.isProvided())
+    {
+        output_monitor.visible = false;
+    }
+}
```

Comment:
This is a wrong way to set default value for params

Reply:
Apparently output_monitor("output_monitor", LLOutputMonitorCtrl::Params().with(...)), but since it's apparently internal element and not a simple value, it should be defined in xml.

Reply:
Reworked